tests/integration/test_content_manifest.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import utils
+import pytest
```

Comment:
In the code base we order imports in three sections: standard library, third-party, and then local imports
```suggestion
import pytest

import utils
```

Reply:
Done! Thank you, I will remember it.